README.md

```diff
@@ -192,6 +192,21 @@ Default: `false` (as of 3.0.0)
 
 Whether or not consent should be implied if the user interacts with the website (clicks anywhere outside the consent manager banner or dialogs).
 
+##### defaultDestinationBehavior
+
+Type: `string`<br>
+Default: 'disable'
```

Comment:
This is the current behavior, prior to PR, correct? So non-breaking change? :)

Reply:
Yeah! Not a breaking change - defaults to the original behavior (not enabling new destinations) 😄